.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+**Description**: Describe the overall goals of the pull request's commits. e.g. *Add a pull request template to the project.*
+
+**JIRA**: [CM-111](http://jira.caremessage.org/browse/CM-111)
+
+**How to Verify**: Describe how the changes can be verified. e.g. *Check if the template has all important information that needs to be described in a pull request.*
+
+**Notes**: Additional notes that should be taken in account. e.g. *All projects must have the pull request template.*
+
+DELETE ONE OF THESE TASK LISTS
+**SIMPLE PR TASKS**
+- [ ] Pick at least one reviewer
+- [x] Merge to *Staging*: See [test, It trys to merge to staging and see if the action works](https://github.com/CareMessagePlatform/check_if_merged/blob/master/.github/workflows/check_if_merged.yml)
```

Comment:
This link will be true when this PR is merged.

For now, it this file: https://github.com/CareMessagePlatform/check_if_merged/pull/3/files#diff-195a635ad245ded9076330e31134bd80R12